### PR TITLE
St feature stage one

### PR DESCRIPTION
**Pull request by saveentoor**

### CODE REVIEW

File: client/js/components/StageOne/StageOne.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        <Progress.Bar
+          style={styles.progressBar}
+          progress={0.3}
+          width={200}
+          color={theme.colors.skyBlue}
+        />
+
+        <View style={styles.compoundInterest}>
+          <Text>
+            <Text style={styles.heading1}>Compound interest</Text> is what most
```
Comment (siddharthparmar7) [inline client/js/components/StageOne/StageOne.js]: `<Text>` cannot be a child of `<Text>`. Please make the changes.

File: client/js/components/StageOne/StageOne.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+              source={require(`../../assets/images/illustrations/Circle.png`)}
+            />
+
+            <View style={styles.numberOne}>
+              <Text style={styles.one}>1</Text>
+            </View>
+          </View>
+          <Text style={styles.textOne}>
+            When we are dealing with
+            <Text style={styles.heading1}>Simple Interest</Text> we are only
```
Comment (siddharthparmar7) [inline client/js/components/StageOne/StageOne.js]: same as above.

File: client/js/components/StageOne/StageOne.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            focusing on the original principal and the interest rate.{" "}
```
Comment (siddharthparmar7) [inline client/js/components/StageOne/StageOne.js]: remove `{" "}`

File: client/js/components/StageOne/StageOne.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+              style={styles.circleImg}
+              source={require(`../../assets/images/illustrations/Circle.png`)}
+            />
+
+            <View style={styles.numberOne}>
+              <Text style={styles.one}>2</Text>
+            </View>
+          </View>
+          <Text style={styles.textTwo}>
+            With <Text style={styles.heading1}>Compound Interest</Text> we also
```
Comment (siddharthparmar7) [inline client/js/components/StageOne/StageOne.js]: <Text> cannot be a child of <Text>. Please make the changes.

File: client/js/components/StageOne/StageOne.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+        <View style={styles.cI}>
+          <Text style={styles.simpleVsCompound}>
+            Simple Interest vs Compound Interest
+          </Text>
+
+          <Text style={styles.textOne}>
+            Both loans have the original principal of $100 and an interest rate
+            of 20%. The left loan is
+            <Text style={styles.heading1}>Simple Interest</Text> and the right
```
Comment (siddharthparmar7) [inline client/js/components/StageOne/StageOne.js]: same as above.

File: client/js/components/StageOne/StageOne.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+            is Compound Interest. Let’s see the difference after a couple of
+            years
+          </Text>
+
+          <View style={styles.table}>
+            <View style={{ width: "50%" }}>
```
Comment (siddharthparmar7) [inline client/js/components/StageOne/StageOne.js]: move your styles to styles.js file.

File: client/js/components/StageOne/StageOne.js [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+              <View style={styles.formula}>
+                <Text style={styles.interest}>
+                  SI=P×r×n{"\n"}P=Principal Amount{"\n"}r=Annual Interest Rate
+                  {"\n"}n=Term of Loan, in years ​
+                </Text>
+              </View>
+            </View>
+
+            <View style={{ width: "50%" }}>
```
Comment (siddharthparmar7) [inline client/js/components/StageOne/StageOne.js]: same as above.

File: client/js/components/StageOne/styles.js [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+    padding: "5%"
+  },
+  content: {
+    flexGrow: 1,
+    alignItems: "center",
+    justifyContent: "space-around"
+  },
+  digitalCurrency: {
+    paddingTop: 20,
+    paddingBottom: 20,
```
Comment (siddharthparmar7) [inline client/js/components/StageOne/styles.js:17]: use paddingVertical: 20

File: client/js/components/StageOne/styles.js [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+  progressBar: {
+    marginBottom: 10
+  },
+  heading1: {
+    color: "#25BDD8",
+    fontWeight: "600"
+  },
+  compoundInterest: {
+    marginTop: height * 0.02,
+    marginBottom: height * 0.03
```
Comment (siddharthparmar7) [inline client/js/components/StageOne/styles.js:31]: use marinVerticle: 20

File: client/js/components/StageOne/styles.js [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+  simpleVsCompound: {
+    color: "white",
+    backgroundColor: theme.colors.skyBlue,
+    width: width * 0.9,
+    fontWeight: "600",
+    fontSize: 15,
+    alignItems: "center",
+    textAlign: "center",
+    paddingTop: "2.5%",
+    paddingBottom: "2.5%"
```
Comment (siddharthparmar7) [inline client/js/components/StageOne/styles.js]: use paddingVerticle.

File: client/js/components/StageOne/styles.js [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+    shadowColor: "#000",
+    shadowOpacity: 0.4,
+    shadowColor: "grey",
+    shadowRadius: 2,
+    shadowOffset: { height: 3, width: 0 },
+    shadowRadius: 3
+  },
+  textOne: {
+    paddingTop: "3%",
+    paddingBottom: "8%",
```
Comment (siddharthparmar7) [inline client/js/components/StageOne/styles.js]: use paddingVerticle.

File: client/js/screens/Lessons/LessonLibrary.js [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+      tabIndex: 0
+    };
+  }
+  updateSearch = search => {
+    this.setState({ search });
+  };
+
+  render() {
+    const { search } = this.state;
+    console.log(search);
```
Comment (siddharthparmar7) [inline client/js/screens/Lessons/LessonLibrary.js]: remove this console.log

File: client/js/screens/Lessons/LessonLibrary.js [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+    const result = topics.filter(topic =>
+      topic.name.toLowerCase().includes(search.toLowerCase())
+    );
+
+    console.log(result);
```
Comment (siddharthparmar7) [inline client/js/screens/Lessons/LessonLibrary.js]: same as above.

**Comment by teelau** [comment]
Somethings not right, some of these older commits are already merged and not part of this feature. Can you cherry pick only the latest changes for this feature?

File: client/package.json [hunk 14]
```diff
@@ -17,6 +17,7 @@
     "react-apollo": "^2.5.6",
     "react-final-form": "^6.1.0",
     "react-native": "0.59.9",
+    "react-native-elements": "^1.1.0",
```
Comment (siddharthparmar7) [inline client/package.json]: we do not use react native elements. Please remove this package and use react native components.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the changes. Thank you.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Move your content to a const or a component state and use it from there instead of hard coding in the text component. Thank you.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**